scripts/fetch-uj-records.sh

```diff
@@ -56,4 +56,4 @@ $QUERYGEN -0 --index="$SPLUNK_INDEX" \
     --data output_mode=json \
     --data earliest_time="$QUERY_EARLIEST_TIME" \
     --data latest_time="$QUERY_LATEST_TIME" \
-    --data search=Q
+    --data-urlencode search=Q
```

Comment:
I wonder why we didn't need this before. Also I think this is sending POST requests, so IMO shouldn't need to URL-encode.

Reply:
It's specifically needed to send a `%` char in the splunk query (see the commit description). This use case also happens to be well documented by Splunk: https://docs.splunk.com/Documentation/Splunk/9.1.0/RESTUM/RESTusing

> In most instances, the -d parameter will sufficiently URI-encode a search string. However, if you are having problems logging in to an endpoint or running a search, use the --data-urlencode parameter instead of -d to ensure all characters in the string are URI-encoded.